src/services/main-serviсe.ts

```diff
@@ -0,0 +1,53 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { ArtworkDetails, getArtworksItemsResponse } from '../types/types';
+
+export const getAllItemsAPI = createApi({
+  reducerPath: 'getAllItemsAPI',
```

Comment:
The names for API slices (getAllItemsAPI, getSearchItemsAPI, and getItemAPI) are similar. Ensure clarity and consistency in naming.